packages/vats/src/virtual-purse.js

```diff
@@ -143,6 +145,18 @@ function makeVirtualPurse(vpc, kit) {
       });
       return pmt;
     },
+    claim: (_paymentP, _optAmountShape) => {
+      // TODO implement
+      assert.fail(X`virtual purses do not yet implement claim`);
```

Comment:
We shouldn't commit with this.  At the very least there would need to be an issue, but what are the design issues for this?

Reply:
Since you have a separate review comment on virtual-purse `getRecoverySet` below and I've already deleted `claim`, I'm resolving this.